Aztec/Classes/TextKit/TextView.swift

```diff
@@ -165,12 +168,54 @@ open class TextView: UITextView {
     ///
     public var outputProcessor: Processor?
 
-    // MARK: - Properties: Text Storage
+
+    // MARK: - TextKit Aztec Subclass'ess
```

Comment:
`/s/Subclass'ess/Subclasses`

No need for the apostrophe / single-quote.

Reply:
Thanks!